draft-ietf-quic-load-balancers.md

```diff
@@ -617,11 +621,11 @@ complete server ID.
 
 # Per-connection state
 
-The CID allocation methods QUIC-LB defines require no per-connection state at
-the load balancer. The load balancer can extract the server ID from the
-connection ID of each incoming packet and route that packet accordingly.
+QUIC-LB requires no per-connection state at the load balancer. The load balancer
+can extract the server ID from the connection ID of each incoming packet and
+route that packet accordingly.
 
-However, once a routing decision has been made, the load balancer MAY
+However, once the routing decision has been made, the load balancer MAY
```

Comment:
Not sure that this change is a positive one.

Reply:
OK